.eslintrc.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
 	extends: [
 		'plugin:@wordpress/eslint-plugin/recommended',
 		'prettier',
+		'prettier/react',
```

Comment:
As part of some updates I had to fix prettier to make linting work properly. This extends is necessary to avoid eslint conflicts with prettier